scripts/helpers/migration.ts

```diff
@@ -0,0 +1,102 @@
+const fs = require("fs");
+const path = require("path");
+
+const truffleContract = require("@truffle/contract");
+
+export async function deployAndUpdate(
+    contractName: string,
+    libs: any,
+    params?: any
+) {
+    return await new Promise(async (resolve, reject) => {
```

Comment:
What's the purpose for all these `await new Promise(async`? 

Instead of wrapping them in a promise and use a resolve callback, why we can't just return the `updatedInstance` at the end?

Reply:
While using `fs.readFile()` the function had some issue while returning `updatedInstance` . So, I used promises.